src/status_im2/contexts/onboarding/enable_biometrics/view.cljs

```diff
@@ -37,13 +41,35 @@
        :style                     {:margin-top 12}}
       (i18n/label :t/maybe-later)]]))
 
+(defn enable-biometrics-parallax
+  []
+  [:<>
+   [parallax/video
+    {:layers  (resources/get-parallax-video :biometrics)
+     :stretch 50}]
+   [rn/view
+    [navigation-bar/navigation-bar {:disable-back-button? true}]
+    [page-title]]])
+
+(defn enable-biometrics-simple
+  []
+  [:<>
+   [rn/view
+    [navigation-bar/navigation-bar {:disable-back-button? true}]
+    [page-title]]
+   [rn/view {:style style/page-illustration}
+    [quo/text
+     "Illustration here"]]])
```

Comment:
Is this ok as a raw string?

Reply:
This is just a placeholder for what should be an illustration. Actually I will export what is in Figma and use that as the fallback for now.